packages/components/atoms/f-form-field/stories/FormField.stories.js

```diff
@@ -1,220 +1,166 @@
-        },
-        labelText: {
-            default: text('Label Text', 'Checkbox Label')
-        },
-        labelDescription: {
-            default: text('Label Description', '')
-        },
-        value: {
-            default: text('Value', 'checkboxLabel')
-        },
-        isDisabled: {
-            default: boolean('isDisabled', false)
-        },
-        hasError: {
-            default: boolean('hasError', false)
-        },
-        isGrouped: {
-            default: boolean('isGrouped', false)
-        },
-        fieldSize: {
-            default: select('Field Size', VALID_FIELD_SIZES, DEFAULT_FIELD_SIZE)
-        },
-        shouldShowLabelText: {
-            default: boolean('shouldShowLabelText', true)
-        },
-        dropdownOptions: {
-            default: object('Dropdown Options', [
-                {
-                    disabled: true,
-                    selected: true,
-                    text: 'Disabled option',
-                    value: ''
-                },
-                {
-                    text: 'As soon as possible',
-                    value: '2021-01-01T01:00:00.000Z'
-                },
-                {
-                    text: 'Today in 5 minutes',
-                    value: '2021-01-01T01:05:00.000Z'
-                }
-            ])
-        },
-        labelDetails: {
-            default: text('Label Details', '')
-        },
-        placeholder: {
-            default: text('Placeholder', '')
-        },
-        valueRadio: {
-            default: text('Value', 'radioLabel')
-        },
-        inputType: {
-            default: select('Input Type', VALID_TEXT_INPUT_TYPES)
-        },
-        prefix: {
-            default: text('prefix', '£')
-        },
-        suffix: {
-            default: text('suffix', 'GBP')
-        }
-    },
-    parameters: {
-        notes: 'some documentation here'
+export const FormFieldComponent = (args, { argTypes }) => ({
+    components: {
+        FormField,
+        AllergyNutIcon,
+        DriverIcon
     },
+    props: Object.keys(argTypes),
     template: `
         <div
         class="u-spacingBottom--large storybook-grid storybook-grid-columns--4 storybook-grid-stack--lessThanWide">
             <form-field
-                :label-text="labelText"
-                :label-description="labelDescription"
-                :value="value"
+                :disabled="isDisabled"
                 :has-error="hasError"
-                :is-grouped="isGrouped"
+                label-text="Checkbox"
+                :labelDescription="labelDescription"
                 input-type="checkbox"
-                :disabled="isDisabled"/>
+                value="value-text"/>
+                <div>
+                    <form-field
+                        :disabled="isDisabled"
+                        :has-error="hasError"
+                        label-text="Radio Grouped - 1"
+                        :labelDescription="labelDescription"
```

Comment:
Nit: Please can you use kebab-case for prop names

Reply:
Just copied across from Knob but can do